packages/beacon-node/src/metrics/metrics/lodestar.ts

```diff
@@ -110,6 +110,12 @@ export function createLodestarMetrics(
         help: "Current count of pending items in reqRespBridgeReqCaller data structure",
       }),
     },
+    networkWorkerWireEventsOnMainThreadLatencySec: register.histogram<"eventName">({
```

Comment:
As per discussion I think we should remove `Sec`-suffix from metric variable names, @tuyennhv since you suggested that any objections from your side?